js-exercises/array-cube-root-to-json/arrayCubeRootToJson.js

```diff
@@ -1,3 +1,34 @@
-const arrayCubeRootToJson = arr => {};
+const isNumberValid = num => {
+  if (typeof num === 'number' || (typeof num === 'string' && !isNaN(Number(num)))) {
+    return true;
+  }
+
+  return false;
+};
+
+const isArrayValid = arr => {
+  if (!Array.isArray(arr)) {
+    throw new Error(`Expected an Array of numbers but recieved: ${arr}`);
+  }
+
+  for (const num of arr) {
```

Comment:
We can club this method in main method directly.
Something like 
for (const num of arr) {
    if (num === null || num === false || isNaN(num)) throw new Error();
    numToCuberootMap[num] = Math.cbrt(Number(num));
  }